ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/application_traffic_recognition.schema.yml

```diff
@@ -0,0 +1,202 @@
+        type: dict
+        keys:
+          l4_ports:
+            type: list
+            primary_key: name
+            description: L4 port field-set.
+            items:
+              type: dict
+              keys:
+                name:
+                  type: str
+                  description: L4 port field-set name.
+                port_values:
+                  type: list
+                  items:
+                    type: str
+                    description: |-
+                      Port values or range of port values.
+                      Port values are between 0 and 65535.
+                    pattern: "^\\d+(\\-\\d+)?$"
+          ipv4_prefixes:
+            type: list
+            primary_key: name
+            description: IPv4 prefix field set.
+            items:
+              type: dict
+              keys:
+                name:
+                  type: str
+                  description: IPv4 prefix field-set name.
+                prefix_values:
+                  type: list
+                  items:
+                    type: str
+                    description: IP prefix (ex- 1.2.3.0/24).
+                    pattern: "^\\(?:[0-9]{1,3}\\.){3}[0-9]{1,3}\/[0-9]{1,2}$"
+      applications:
+        type: dict
+        keys:
+          ipv4_applications:
+            type: list
+            description: List of user defined IPv4 applications.
+            primary_key: name
+            items:
+              type: dict
+              keys:
+                name:
+                  type: str
+                  description: Application name.
+                src_prefix_set_name:
+                  type: str
+                  description: Source prefix set name.
+                dest_prefix_set_name:
+                  type: str
+                  description: Destination prefix set name.
+                protocols:
+                  # Using a dict here because of the custom behavior of UDP and TCP
+                  type: list
+                  description: |-
+                    List of protocols to consider for this application.
+
+                    To use port field-sets (source, destination or both), the list
+                    must contain only one protocol.
+                    To set port field-set(s) for both TCP and UDP, use `tcp_udp`.
+                  items:
+                    type: str
+                    valid_values:
+                      - ahp
+                      - esp
+                      - icmp
+                      - igmp
+                      - ospf
+                      - pim
+                      - rsvp
+                      - tcp
+                      - udp
+                      - vrrp
+                protocol_ranges:
+                  type: list
+                  description: |-
+                    Acccept protocol value(s) or range(s).
+                    Protocol values can be between 1 and 255.
+                  items:
+                    type: str
```

Comment:
convert_types

Reply:
We cannot have convert types here as the one can also enter the range values like: 24-26